test/cpp/end2end/xds/xds_end2end_test.cc

```diff
@@ -1177,6 +1174,8 @@ TEST_P(XdsServerSecurityTest,
                                              backends_[0]->port(),
                                              default_server_route_config_);
   backends_[0]->Start();
+  backends_[0]->notifier()->WaitOnServingStatusChange(
```

Comment:
Can we also add a timeout in `WaitOnServingStatusChange()`?  Otherwise, if the update never happens, the test will just hang instead of failing quickly.

Reply:
done